### PR TITLE
Tenant uuid added when retrieving agent options

### PR DESCRIPTION
**Pull request by xxbillouxx**

### CODE REVIEW

File: wazo_agid/objects.py [hunk 1]
```diff
@@ -579,52 +579,63 @@ def pickupgroups(self):
...
-            )
-        else:
-            raise LookupError("id or number must be provided to look up an agent")
+    @classmethod
+    def from_number(cls, cursor: DictCursor, number: str, tenant_uuid: str) -> Agent:
+        query = SQL(
+            "SELECT {columns} FROM agentfeatures WHERE {field} = %s and tenant_uuid = %s"
+        ).format(
+            columns=join_column_names(cls._columns),
+            field=Identifier('number'),
```
Comment (fblackburn1) [inline wazo_agid/objects.py:627]: You could write `number` directly inside the query

File: wazo_agid/objects.py [hunk 2]
```diff
@@ -579,52 +579,63 @@ def pickupgroups(self):
...
+        self.firstname = firstname
+        self.lastname = lastname
+        self.language = language
+        self.preprocess_subroutine = preprocess_subroutine
+
+    @classmethod
+    def from_id(cls, cursor: DictCursor, agent_id: str) -> Agent:
+        query = SQL("SELECT {columns} FROM agentfeatures WHERE {field} = %s").format(
+            columns=join_column_names(cls._columns),
+            field=Identifier('id'),
```
Comment (fblackburn1) [inline wazo_agid/objects.py:617]: You could write `id` directly inside the query

File: integration_tests/suite/test_handlers.py [hunk 3]
```diff
@@ -110,6 +110,18 @@ def test_agent_get_options(base_asset: BaseAssetLaunchingHelper):
...
+        agent['tenant_uuid'],
+        f'*{agent["id"]}',
+    )
+
+    assert recv_cmds['FAILURE'] is False
+    assert recv_vars['XIVO_AGENTEXISTS'] == '1'
+    assert recv_vars['XIVO_AGENTPASSWD'] == ''
+    assert recv_vars['XIVO_AGENTID'] == str(agent['id'])
+    assert recv_vars['XIVO_AGENTNUM'] == agent['number']
+    assert recv_vars['CHANNEL(language)'] == agent['language']
```
Comment (fblackburn1) [inline integration_tests/suite/test_handlers.py:123]: Love to only see this diff to add a test :heart:

**Review by fblackburn1 — COMMENTED** [review]
(I don't have time to review, just clicking on the PR by curiosity)

**Review by sduthil — APPROVED** [review]

File: integration_tests/suite/test_handlers.py (hunk 3 above)
Comment (sduthil) [reply integration_tests/suite/test_handlers.py:123]: The test has been added in that PR: https://github.com/wazo-platform/wazo-agid/pull/136

**Pull request merged**